### PR TITLE
Update to use ES6 modules

### PR DESCRIPTION
**Pull request by 0xGabi**
# 🦅 Pull Request

Build on #950 after refactor new package.
Add `aragen` commands to `aragon-cli`.

### CODE REVIEW

File: packages/aragon-cli/src/commands/dao_cmds/utils/getRepoTask.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { DEFAULT_IPFS_TIMEOUT } from '@aragon/toolkit/dist/helpers/constants'
 //
 const LATEST_VERSION = 'latest'
 
-module.exports = {
+export default {
```
Comment (dapplion) [inline packages/aragon-cli/src/commands/dao_cmds/utils/getRepoTask.js]: **Todo**: Replace 
```js
export default { funcA, funcB }
```
with
```js
export funcA
export funcB
```
**And** make sure other modules importing it are compatible with the new syntax

File: packages/aragon-cli/src/commands/devchain_cmds/utils/ganache-vars.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default {
```
Comment (dapplion) [inline packages/aragon-cli/src/commands/devchain_cmds/utils/ganache-vars.js]: **Todo**: Replace 
```js
export default { funcA, funcB }
```
with
```js
export funcA
export funcB
```
**And** make sure other modules importing it are compatible with the new syntax

**Comment by dapplion** [comment]
@0xGabi Thanks for the quick fix of the suggested changes!

I propose a small improvement in the naming of the functions now that we are editing these files, for future clarity https://github.com/aragon/aragon-cli/pull/1007

File: packages/aragon-cli/package.json [hunk 3]
```diff
@@ -1,188 +1,189 @@
...
-    ],
-    "parser": "babel-eslint"
-  },
-  "prettier": {
-    "singleQuote": true,
-    "semi": false,
-    "trailingComma": "es5",
-    "bracketSpacing": true
-  }
+	"name": "@aragon/cli",
```
Comment (0x6431346e) [inline packages/aragon-cli/package.json]: Any reason for converting this spaces into tabs?

Comment (0xGabi) [reply packages/aragon-cli/package.json]: Mmm no,we should update that 😅

**Pull request merged**